### ISSUE TITLE
Castle construction screen, shadow of the text in the information bar at the bottom of the screen, the shadow of the text should not be displayed over the yellow frame of the info bar

### ISSUE THREAD

**Issue by LeHerosInconnu** [ihhub/fheroes2#1745]
In fheroes2, the shadow of the text is displayed over the yellow frame of the info bar.
In the original game, this is not the case.

In original game:

![Building sign position H2 01](https://user-images.githubusercontent.com/43583994/93666264-177ac800-fa7d-11ea-8962-05cebe0a0628.png)

In fheroes2:

![Building sign position fh2 01](https://user-images.githubusercontent.com/43583994/93666266-1e093f80-fa7d-11ea-8037-1336d691db65.png)

Similar issues: https://github.com/ihhub/fheroes2/issues/1675, https://github.com/ihhub/fheroes2/issues/1728.

**Comment by Branikolog** [comment]
Isn't it the same issue you talked about here #1728?

**Comment by LeHerosInconnu** [comment]
@Branikolog.

> Isn't it the same issue you talked about here #1728?

Yes, it's the same problem but in a different window.
I would group the same type of issues in the same thread in the future.

**Comment by Branikolog** [comment]
Should I close this for now?

**Comment by LeHerosInconnu** [comment]
> Should I close this for now?

Why? Are these two windows managed by the same part of the code?

**Comment by Branikolog** [comment]
Yes. information bar for all of this windows is the same, as well, as text font. So, the issues regarding font shadows, are part of the same code.

**Comment by LeHerosInconnu** [comment]
@Branikolog.

> Yes. information bar for all of this windows is the same, as well, as text font. So, the issues regarding font shadows, are part of the same code.

Do what you think is best to get the job done right. :)

**Comment by Branikolog** [comment]
It's ok! :)
You can always ask me in discord for the cases, you're not sure about. Or here, at a similar topic, if you find one.

**Comment by LeHerosInconnu** [comment]
@Branikolog.

> It's ok! :)
> You can always ask me in discord for the cases, you're not sure about. Or here, at a similar topic, if you find one.

Okay, thank you.

**Comment by zenseii** [comment]
The text just needs to be shifted up one pixel:
![image](https://user-images.githubusercontent.com/12501091/190876904-9ca3289b-9098-47a7-a41d-67fd563d6918.png)

**Comment by LeHerosInconnu** [comment]
Hello @zenseii,

The text just needs to be shifted up one pixel:
![image](https://user-images.githubusercontent.com/12501091/190876904-9ca3289b-9098-47a7-a41d-67fd563d6918.png)

I think that it will unbalance the visual in the case of a text with no letters with a jamb (letters of the Latin alphabet with a jamb:
" g ", " j ", " p ", " q " et " y ").
Better not to display the shadow of the letters with jamb over the yellow frame, which is the case in the original game for some screens.

In original game:

![No shadow over yellow frame letter with jamb 01](https://user-images.githubusercontent.com/43583994/190919593-57ea21b9-4934-4c45-bbf2-262b04bffb7a.png)
